rpc/backend/tx_info.go

```diff
@@ -66,9 +68,12 @@ func (b *Backend) getRPCTransactionForEndBlockTx(txDetails *EndBlockTxDetails) (
 		Type:             2,
 		Accesses:         nil,
 		ChainID:          (*hexutil.Big)(b.chainID),
-		V:                nil,
-		R:                nil,
-		S:                nil,
+		// V:                nil,
```

Comment:
can we remove these comments?
what is difference between having nil and 0 here?

Reply:
yes. this was ST geth could unmarshal tx objects on client. before it would error out. I can add a comment explaining